architecture/additional_concepts/authentication.adoc

```diff
@@ -697,6 +697,69 @@ responses from `/oauth/authorize`, and how they should be handled:
 |Other  | Other                                                                                                                                            | Fail, optionally surfacing response body to the user
 |===
 
+For example, 
```

Comment:
Since we have a table between the introduction of the action and the example, I'd do something like s/For example,/To request an OAuth token using the implicit grant flow: